data/tipsOfTheDay.ts

```diff
@@ -0,0 +1,51 @@
+export interface ITips {
+	title: string
+	description: string
+	tipLink: string
+}
+
+export const TIPS: ITips[] = [
```

Comment:
Please use lowercase

Reply:
As I was using dummy content now I kept it in uppercase but I will change to lowercase now